### PR TITLE
Feature/milestone 5

### PR DESCRIPTION
**Pull request by defoebrand**
This milestone:
 - Builds App as class with constructor and state
 - Renders display and button panel
 - Updates display with state change
 - Wires logic to click event on button
 - Passes click event defined in App to Button through ButtonPanel

### CODE REVIEW

**Review by Torres-ssf — APPROVED** [review]
Review in Process

File: src/components/App.js [hunk 1]
```diff
@@ -5,13 +5,49 @@ import ButtonPanel from './ButtonPanel';
...
+        display = <Display result={`${next}`} />;
+      } else {
+        display = <Display result="Hello!" />;
+      }
+      if (operation) {
+        display = <Display result={`${next} ${operation}`} />;
+      }
+      if (total) {
+        display = <Display result={`${total} ${operation} ${next}`} />;
+      }
```
Comment (Torres-ssf) [inline src/components/App.js]: - [x] We need to find a way to improve this logic here. Right now, if all states are different from null, the display variable will be reassigned 3 times.

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi @defoebrand :hand: 
Nice job on this milestone so far. But we have changes to make before moving on. :muscle: 

- [x] Right now there is a bug in the application that is causing the operations to not work as expected.
bug1
After clicking at the equal sign the application crashes.

**OPTIONAL**

- [x] There is no requirement for styling this application but I strongly suggest you apply some styles. It does not need to be anything fancy yet. But for now, we can organize the buttons to have the same order that calculator buttons have

Please feel free to message me at slack if you have any questions about this project :smile: 
Happy coding :computer: 
**Sergio Torres**

File: src/components/App.js (hunk 1 above)
Comment (defoebrand) [reply src/components/App.js]: I have addressed the bug and believe it is gone, the page has rudimentary styling to group buttons appropriately, and the display logic is cleaned up.

**Review by rOluochKe — APPROVED** [review]
Hello @defoebrand 
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**